src/rendering/texformats.jl

```diff
@@ -185,7 +198,7 @@ function format_chunk(chunk::DocChunk, docformat::JMarkdown2tex)
     return docformat.keep_unicode ? out : uc2tex(out)
 end
 
-function format_output(result, docformat::JMarkdown2tex)
+function format_output(result, docformat::TexFormat)
```

Comment:
I think this will produce "invalid" output `texminted` format, right ?
Hightlights.jl will include its own characters around some special characters e.g. `_`, so that it will result in weird output for minted output.
How about defining new `format_output` for `TexMinted` ?

Reply:
Well spotted. I hadn't figured this out.

Reply:
fixed it.
However it did make my code have more duplicate lines again.